pyproject.toml

```diff
@@ -1,20 +1,82 @@
-[tool.codespell]
-builtin = "clear,rare,informal,names,usage"
-ignore-words = "ignore_words.txt"
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [
+  "setuptools",
+  "setuptools-scm",
+]
+
+[project]
+name = "sphinx-gallery"
+description = "A Sphinx extension that builds an HTML gallery of examples from any set of Python scripts."
+readme = "README.rst"
+license = {"text" = '3-clause BSD'}
+authors = [
+   {name = "Óscar Nájera", email = 'najera.oscar@gmail.com'},
+]
+requires-python = '>=3.8'
+classifiers = [
+  'Development Status :: 4 - Beta',
+  'Framework :: Sphinx :: Extension',
+  'Intended Audience :: Developers',
+  'Programming Language :: Python',
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+]
+dynamic = [
+  "version",
+]
+dependencies = [
```

Comment:
Looks like a little bit of unintentional information loss -- can you copy over the comment from `requirements.txt` about needing to update the README when something changes here?

Reply:
adding in 1b69dce87c8af4ec638ca8e5dd4ac9385794b9bb